iceoryx_dds/include/iceoryx_dds/gateway/gateway_config.hpp

```diff
@@ -32,7 +32,7 @@ struct GatewayConfig
     {
         iox::capro::ServiceDescription m_serviceDescription;
     };
-    iox::cxx::vector<ServiceEntry, iox::MAX_PORT_NUMBER> m_configuredServices;
+    iox::cxx::vector<ServiceEntry, iox::MAX_PUBLISHERS + iox::MAX_SUBSCRIBERS> m_configuredServices; 
```

Comment:
how about putting this somewhere as e.g. `iox::MAX_GATEWAY_SERVICES`?

Reply:
forwarding this question to @ithier

Reply:
We could add it to `iceoryx_posh_types.hpp` since once #227 is merged the gateway building blocks will also be part of posh.